### PR TITLE
Remote ID in PX4 v1.14

### PR DESCRIPTION
**Pull request by hamishwillee**
This adds a page from Remote ID in PX4 v1.14. I wanted to capture that it is experimental, and exactly what you can expect right now.

I only added Cube ID as that is the only system I have had confirmed works with current main.

THoughts?

@mrpollo @junwoo091400 et al

PS Am requesting confirmation of Cube ID setup on https://discord.com/channels/1022170275984457759/1108437678871019530/1128846440702169188

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -317,6 +317,7 @@
       * [Servo Gripper](peripherals/gripper_servo.md)
     * [Parachute](peripherals/parachute.md)
     * [ADSB/FLARM (Traffic Avoidance)](peripherals/adsb_flarm.md)
+    * [Remote ID](peripherals/remote_id.md.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:320]:
```suggestion
    * [Remote ID](peripherals/remote_id.md)
```

File: en/peripherals/remote_id.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+- [OPEN_DRONE_ID_AUTHENTICATION](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_AUTHENTICATION) - Provides authentication data for the UAV.
+- [OPEN_DRONE_ID_SELF_ID](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_SELF_ID) - Operator identity (plain text).
+- [OPEN_DRONE_ID_OPERATOR_ID](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_OPERATOR_ID) - Operator identity.
+- [OPEN_DRONE_ID_ARM_STATUS](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_ARM_STATUS) - Status of Remote ID hardware. Use as condition for vehicle arming, and for Remote ID health check.
+- [OPEN_DRONE_ID_SYSTEM_UPDATE](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_SYSTEM_UPDATE) - Subset of `OPEN_DRONE_ID_SYSTEM` that can be sent with information at higher rate.
+
+### Compliance
+
+The specific areas where PX4 v1.14 is know to (possibly) be non compliant with various specifications are:
```
Comment (junwoo091400) [inline en/peripherals/remote_id.md:116]:
```suggestion
The specific areas where PX4 v1.14 is known to (possibly) be non compliant with various specifications are:
```

File: en/peripherals/remote_id.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+You will now find a new parameter called [SER_TEL2_BAUD](../advanced_config/parameter_reference.md#SER_TEL2_BAUD).
+The required baud rate depends on the remote ID used (for Cube ID it must be set to 57600).
+
+<!-- In theory, a Remote ID (or other MAVLink peripheral) that supports WiFi or wired Ethernet network could also be connected over those links. -->
+
+
+### Enable Remote ID
+
+Enable Remote ID by [setting](../advanced_config/parameters.md#conditional-parameters) parameter [COM_ARM_ODID](#COM_ARM_ODID) to `2` (it is disabled by default).
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/peripherals/remote_id.md:78]: Thanks, I'll change it.

File: en/peripherals/remote_id.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+<a id="COM_ARM_ODID"></a>[COM_ARM_ODID](../advanced_config/parameter_reference.md#COM_ARM_ODID)  | Enable Drone ID system detection and health check. `0`: Disable (default), `1`: Warn if Remote ID not detected but still allow arming, `2`: Only allow arming if Remote ID is present.
+
+
+## Implementation
+
+Remote ID is enabled by setting parameter [COM_ARM_ODID](../advanced_config/parameter_reference.md#COM_ARM_ODID) to `2`.
+The UAV will then require `HEARTBEAT` messages from the Remote ID as a precondition for arming the UAV.
+You can also set the parameter to `1` to warn but still allow arming without the Remote ID.
+
+Once enabled, PX4 v1.14 streams these messages by default (in streaming modes: normal, onboard, usb, onboard low bandwidth):
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/peripherals/remote_id.md:94]: We certainly don't want to stream stuff if it is not needed. I know we have a pattern for streaming by default, and for requesting a stream. Do we have a pattern for enabling streaming based on a parameter or enabling a particular driver?

Most places we fly are going to need this, so default enabling probably not the end of the world.

File: en/peripherals/remote_id.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+- [OPEN_DRONE_ID_LOCATION](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_LOCATION) (1 Hz) - UAV location, altitude, direction, and speed.
+- [OPEN_DRONE_ID_SYSTEM](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_SYSTEM) (1 Hz) Operator location/altitude, multiple aircraft information (group/swarm, if applicable), full timestamp and possible category/class information.
+  - Implementation assumes operator is located at vehicle home position (does not yet support getting operator position from GCS).
+    This is believed to be compliant for broadcast-only Remote IDs.
+  
+The following message can be streamed on request (using [MAV_CMD_SET_MESSAGE_INTERVAL](https://mavlink.io/en/messages/common.html#MAV_CMD_SET_MESSAGE_INTERVAL)):
+
+- [OPEN_DRONE_ID_BASIC_ID](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_BASIC_ID) - UAV identity information (essentially a serial number)
+  - PX4 v1.14 specifies a serial number ([MAV_ODID_ID_TYPE_SERIAL_NUMBER](https://mavlink.io/en/messages/common.html#MAV_ODID_ID_TYPE_SERIAL_NUMBER)) but does not use the specified format (ANSI/CTA-2063 format).
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/peripherals/remote_id.md:104]: You said it wouldn't, and I believe you.

File: en/peripherals/remote_id.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/peripherals/remote_id.md]: I have updated docs appropriately

**Comment by hamishwillee** [comment]
@junwoo091400 Thank you for your review. I have updated so it is now accurate for current state of play. Of course if https://github.com/PX4/PX4-Autopilot/pull/21647 goes in then it would require an update. 

Merging now.

**Pull request merged**